MAUI/Picker/getting-started.md

```diff
@@ -0,0 +1,308 @@
+---
+layout: post
+title: Getting started with .NET MAUI Picker control | Syncfusion
+description: Learn here all about getting started with Syncfusion .NET MAUI Picker (SfPicker) control and its basic features.
+platform: maui
+control: SfPicker
+documentation: ug
+---
+
+# Getting started with .NET MAUI Picker (SfPicker)
+This section explains how to add the Picker control. This section covers only the basic features needed to get started with Syncfusion Picker.
+
+## Creating an application using the .NET MAUI Picker
+
+1. Create a new .NET MAUI application in Visual Studio.
+
+2. Syncfusion .NET MAUI components are available on [nuget.org](https://www.nuget.org/). To add `SfPicker` to your project, open the NuGet package manager in Visual Studio, search for [Syncfusion.Maui.Picker](https://www.nuget.org/packages/Syncfusion.Maui.Picker), and then install it.
```

Comment:
2. Syncfusion .NET MAUI components are available on [nuget.org](https://www.nuget.org/). To add `SfPicker` to your project, open the NuGet package manager in Visual Studio, search for [Syncfusion.Maui.Picker](https://www.nuget.org/packages/Syncfusion.Maui.Picker), then install it.

Reply:
Content modified.